dictionary.json

```
@@ -11474,6 +11914,20 @@
     "examples": [],
     "fields": []
   },
+  {
+    "toaq": "meıme",
+    "type": "predicate",
+    "english": "▯ is the mountains.",
+    "gloss": "mountains",
```

I don't think "the" is baked into this word. 
```suggestion
    "english": "▯ are mountains.",
    "gloss": "mountains",
```
Or maybe:
```suggestion
    "english": "▯ is a mountain range.",
    "gloss": "mountain.range",
```

---

Yeah, I think "mountain range" works the best here, thanks for that one. "Are mountains" sounds like a redefinition of **meı** at least to my ear.